docs/DeveloperGuide.adoc

```diff
@@ -721,20 +721,67 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 |`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
 
-|`* *` |user |hide link:#private-contact-detail[private contact details] by default |minimize chance of someone else seeing them by accident
+|`* * *` |user |use acronyms instead of the full command |use the app more efficiently
 
-|`*` |user with many persons in the address book |sort persons by name |locate a person easily
-|=======================================================================
+|`* * *` |user |view all entries in alphabetical order |easily find the contact I'm looking for
+
+|`* * *` |user |store birthdays of my contacts |remember them
+
+|`* * *` |user |import contacts from Google Contacts/iCloud |populate the app without having to add manually
+
+|`* * *` |user |email a contact by clicking his/her email or shortcut |talk to people faster
+
+|`* * *` |user |view the home address of a contact in Google Maps |get directions quicker
+
+|`* * *` |experienced user |remove duplicate contacts |have a cleaner Address Book
+
+|`* * *` |user |view my favourite contacts immediately |save time by not having to find them
+
+|`* * *` |user |store my contacts' social media usernames |easily access their profiles
+
+|`* * *` |lazy user |access stored social media accounts of my contacts by clicking/shortcut |access their profiles quicker
+
+|`* * *` |user |list contacts by their initials [A...Z] to search for anyone with names starting with [A...Z] |find contacts even though I do not remember their exact names
+
+|`* * *` |sociable user |export a contact |share their details with other friends
+
+|`* * *` |power user |define my own alias for the shortcuts |use the application more efficiently
+
+|`* *` |user |add display picture for my contact |can easily identify them
+
+|`* *` |sociable user |list all the persons having same birthday |will not miss out on my friends' birthday
+
+|`* *` |user |increase the font size of the address book |see more clearly
+
+|`* *` |experienced user |able to see the number of times I interacted with a contact |know who I frequently contact
+
+|`* *` |experienced user | view the history of commands in a chronological order | see what command I entered at the start
+
+|`* *` |new user | enter commands in a human-friendly manner | use the application in a more flexible manner
+
+|`* *` |security-conscious user | secure my application using a password | ensure that only I can access the data
 
-{More to be added}
+|`* *` |forgetful user | add reminders that appear during launch | remember my meetings with people
+
+|`* *` |experienced user | send emails to a selected group of contacts | contact them all at once
+
+|`* *` |experienced user | search by tags | easily find the person I'm looking for
+
+|`*` |security-conscious user | encrypt the application's data file | ensure that my contacts' data is safe
+
+|`*` | user | view my recent contacts | remember who I last contacted
```

Comment:
Another extra 'space' in front of 'user' word here.

Pay attention to format and be consistent. Some of your words start right after `|`, some has a space after `|` before the words. --> Relook at your user stories, many of them are inconsistent. (Discuss and enforce formats to make the work seem like it is done by only 1 person and not many!)